modules/network/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "region" {
+    description = "AWS Region"
+    type = string
+    default = "us-east-2"
+}
+
+variable "availability_zones" {
+    description = "Availability Zones"
+    type        = list(string)
+    default     = ["us-east-2a", "us-east-2b", "us-east-2c"]
+}
+
+variable "application_name" {
+    description = "Set application name(eg. prod, dev.etc)"
+    type        = string
+    default     = "example-app"
+}
+
+variable "vpc_cidr" {
+    type = string
+    default = "10.0.0.0/16"
+}
+
+# variable "vpc_id" { 
+#     type = string
+# }
+
+variable "default_route" {
+    description = "Default Route from and to the internet"
+    type        = string
+    default     = "0.0.0.0/0"
+}
+
+variable "tenancy" {
+    default = "default"
+}
+
+variable "true" {
```

Comment:
i dont think you need to have a variable for something that will not change. if this value will change, then give it a better name than "true"

Reply:
Since the true value can be specified inline and does not change, I took out the true variable.